release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -346,6 +346,11 @@ If you are using the OVN-Kubernetes network plugin, you can migrate to the OpenS
 
 For more information, see xref:../networking/openshift_sdn/migrate-to-openshift-sdn.adoc#migrate-to-openshift-sdn[Migrating to the OpenShift SDN network plugin].
 
+[id="ocp-4-13-core-dns-update"]
+==== CoreDNS updated to 1.10.1
+
+{product-title} 4.13 4.13 updates CoreDNS to 1.10.1. CoreDNS now uses the DNSSEC DO Bit that was specified on originating client query. This reduces the DNS response UDP packet size when a client is not requesting DNSSEC. Consequently, the smaller packet size reduces the chance of DNS truncation decreasing by TCP connection retries as well as overall DNS bandwidth.
```

Comment:
change to `{product-version}` attribute and a couple of minor edits to help with sentence clarity
```suggestion
{product-title} {product-version} updates CoreDNS to 1.10.1. CoreDNS now uses the DNSSEC DO Bit that was specified on the originating client query. This reduces the DNS response UDP packet size when a client is not requesting DNSSEC. Consequently, the smaller packet size reduces both the chance of DNS truncation decreasing by TCP connection retries as well as overall DNS bandwidth.
```